### PR TITLE
Backport 14941, 15149 and 15210

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Intended for merge via merge commit

### CODE REVIEW

**Review by UdjinM6 — COMMENTED** [review]
LGTM but you might want to also backport 15210 while at it

**Comment by PastaPastaPasta** [comment]
Happy now!?! :D

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Almost :)

File: src/qt/bitcoingui.cpp [hunk 1]
```diff
@@ -1712,11 +1712,13 @@ void BitcoinGUI::updateWindowTitle()
             window_title += userWindowTitle + " - ";
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/qt/bitcoingui.cpp]: done, can squash if you want

**Comment by UdjinM6** [comment]
Looks good now :) pls squash

**Comment by PastaPastaPasta** [comment]
Squashed

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**